server/server.go

```diff
@@ -0,0 +1,62 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/medibloc/panacea-doracle/config"
+	"github.com/medibloc/panacea-doracle/server/service"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func Run(conf *config.Config) error {
```

Comment:
As I remember, we don't need this HTTP server anymore. Instead, we are gonna run a Tendermint event subscriber. Because the subscriber will be implement in other PRs, it's enough for you to make this blank.

Reply:
Well, we will probably need to have a HTTP endpoint or so for some admin purposes, but not sure for now.

Reply:
As you said and we discussed off-line, we decided to keep use HTTP server for some future admin purpose.